src/containers/ClassifierContainer.jsx

```diff
@@ -62,6 +65,14 @@ class ClassifierContainer extends React.Component {
     if (nextProps.currentSubject && nextProps.workflow && !this.props.classification) {
       this.props.dispatch(createClassification());
     }
+
+    if (nextProps.workflow && nextProps.preferences && nextProps.tutorialStatus === TUTORIAL_STATUS.IDLE) {
+      this.props.dispatch(fetchTutorial(nextProps.workflow));
+    }
+
+    if (nextProps.tutorial !== this.props.tutorial) {
+      Tutorial.startIfNecessary(Tutorial, nextProps.tutorial, nextProps.user, this.props.preferences);
```

Comment:
Two things:
1. Is there a reason this uses `this.props.preferences` instead of `nextProps.preferences`?
2. Does the _if-clause_ here need to check for the existence of nextProps.tutorial/user/preferences? I'm unfamiliar with the mechanisms of ZRC's Tutorial, alas.

Reply:
Glad you're bringing this up because I did a good bit of modifying with `this.props` vs. `nextProps`. 

1) Yes, this should probably be `nextProps`.
2) I don't think it needs to check the presence of `user` or `preferences` on this line because it is checked a few lines above for the initial tutorial fetch. A tutorial will not be fetched without preferences or a user, so that `if (nextProps.tutorial !== this.props.tutorial)` line _shouldn't_ be reached unless a user or preferences was already available.